adapter/repository/in-memory-simple/src/main/java/org/mobilitydata/gtfsvalidator/db/InMemoryExecParamRepository.java

```diff
@@ -258,9 +259,6 @@ public Options getOptions() {
                     option.setArgs(0);
                     break;
                 }
-                case EXCLUSION_KEY: {
```

Comment:
I am not sure to understand why that had to be removed. It seems the EXCLUSION_KEY gets a very specific treatment now, or it it because UNLIMITED_VALUES is the default?

Reply:
This had to be removed since EXCLUSION_KEY does not longer needs a specific treatment. As all options, it can only be defined once with an unique argument(which is a  String such as `agency.txt,route.txt`